tests/test_oidc.py

```diff
@@ -0,0 +1,138 @@
+        '8cs4iw4u8QSEOSEzL5Gy1wXPdL78aoaqUYg'),
+    'id_token': (
+        'eyJhbGciOiJSUzI1NiIsImtpZCI6IjRmYjE1NjVlYWRlNTFiOWMzYTUyYmU0ND'
+        'I0YjNjYTkxYzM4ZjUzNjUiLCJ0eXAiOiJKV1QifQ.eyJhdWQiOiI4cHJZb1p5bTB6a3Q2'
+        'WnpUSWJYYUZCUXZ0cldpMkE4eG1kQzlXRXozaTE0PUBsb2NhbGhvc3QiLCJlbWFpbCI6I'
+        'mtlbi50c2FuZ0BkaWdpdGFsLmNhYmluZXQtb2ZmaWNlLmdvdi51ayIsImVtYWlsX3Zlcm'
+        'lmaWVkIjp0cnVlLCJleHAiOjE0NzM4OTYzNjYsImlhdCI6MTQ3Mzg1MzE2NiwiaXNzIjo'
+        'iaHR0cDovL2RleC5leGFtcGxlLmNvbTo1NTU2IiwibmFtZSI6IiIsInN1YiI6IjZjN2E5'
+        'ZjQ1LTdmNTctNGQ5MS1iZTBlLTI4NjY3M2EyOGM2ZiJ9.tAVC2OD70vuTiARWoSagm37x'
+        'QcWZ3o8W9jLvW8mHG39MgOp6GHGhyJuTgvkciDqi10SqHMcaGH9jSZepVUkQBNYPKejp9'
+        'VZ3iiXyq731ckzoY93q5TvSOqjkoG7_HxXCkD5RX2F6XdTq_Se231TSEgWPxYl3ycLzKt'
+        'NMeD5o3Aq8z_ypzgl7kQmEEdZWPSAcQr7-6IIHJ38UgDZfPhTYtUB4f_abgXXcuQV10uW'
+        'kXBMdOzfM2s9ByexSAvL2-HVs_jtdC3C-Rwu_05yKfduVO5yiNBxoyrkv2yZgEhfKNh1W'
+        'LYj2cb08cs4iw4u8QSEOSEzL5Gy1wXPdL78aoaqUYg'),
+    'token_type': 'bearer'}
+
+
+@pytest.fixture
+def claims(utcnow):
+    config = {
+        "issuer": "http://dex.example.com:5556",
+        "authorization_endpoint": "http://dex.example.com:5556/auth",
+        "token_endpoint": "http://dex.example.com:5556/token",
+        "jwks_uri": "http://dex.example.com:5556/keys",
+        "response_types_supported": [
+            "code"
+        ],
+        "grant_types_supported": [
+            "authorization_code",
+            "client_credentials"
+        ],
+        "subject_types_supported": [
+            "public"
+        ],
+        "id_token_signing_alg_values_supported": [
+            "RS256"
+        ],
+        "token_endpoint_auth_methods_supported": [
+            "client_secret_basic"
+        ],
+        "redirect_uri": "test",
+        "client_id":
+        "8prYoZym0zkt6ZzTIbXaFBQvtrWi2A8xmdC9WEz3i14=@localhost",
+            "client_secret": "testsecret",
+            "keys": [
+                {
+                    "kid": "4fb1565eade51b9c3a52be4424b3ca91c38f5365",
+                    "kty": "RSA",
+                    "alg": "RS256",
+                    "use": "sig",
+                    "e": "AQAB",
+                    "n": (
+                        "zzJAqac1b3GVEehBZUp48gjJpXkD-vGsFyeTmWJGl00NVmgimW-h7"
+                        "UUrVtJiwnaXS98D3ZPFnQRUrn1bmLeADkHs2sbWyfCNTYVK2sBY0Y"
+                        "K6AamIwCLEET8aWB2b0P6L7xpM8lP8y3ss42ICgWV73txv-KwI8Fw"
+                        "aDScIzSzSSdWWgqxh54PCMGAbWJCg6kjyufBocBVaZLUL_4HKIixI"
+                        "SgyThUpFYND8iaQThI6fkW5o3Qc95AEcm5XKvlEXYv-BWWj-xBXka"
+                        "FDy9YOjUKAXcYmWPhbY5LGM1GYk79PigNggxgRDsrxWDNQUzH25o8"
+                        "wxRfOX1cZ-bdVzGYHDYeUbpw=="
+                    )
+                }
+        ]
+    }
+    now = datetime.datetime(2016, 9, 14, 12, 40)
+    with utcnow("jose.jwt.datetime", now):
+        claims = verify_id_token(test_token_response['id_token'], config)
```

Comment:
Since we're only using `test_token_reponse['id_token']`, we can discard all the other `test_token_response` values, and just have an `id_token` variable.